lib/config.js

```diff
@@ -6,10 +6,25 @@
 const bedrock = require('bedrock');
 const {config} = bedrock;
 const path = require('path');
+const {Ed25519VerificationKey2020} =
+  require('@digitalbazaar/ed25519-verification-key-2020');
+const {CryptoLD} = require('crypto-ld');
+
+const cryptoLd = new CryptoLD();
+cryptoLd.use(Ed25519VerificationKey2020);
 
 const namespace = 'vc-issuer';
 const cfg = config[namespace] = {};
 
+// bedrock-did-io configuration
+config['did-io'].methods.key.verificationSuite = Ed25519VerificationKey2020;
+
+config['did-io'].methods.v1.verificationSuite = Ed25519VerificationKey2020;
+config['did-io'].methods.v1.cryptoLd = cryptoLd;
```

Comment:
I suspect we want to set all of these things at the app level, not here. This creates a tight binding and introduces other dependencies that make this module harder to maintain.